DasharoModulePkg/UserAuthenticationDxe/UserAuthenticationDxe.c

```diff
@@ -173,7 +173,7 @@ PrintSetPasswordStatus (
 
   if (ReturnStatus == EFI_UNSUPPORTED) {
     DisplayString  = L"New password is not strong enough!";
-    DisplayString2 = L"Password must at least 8 chars and include lowercase, uppercase alphabetic, number and symbol";
+    DisplayString2 = L"Password must at least 8 chars";

```

Comment:
must ... - isn;t something missing here?

Reply:
Right, someone made a typo here before, might as well fix it

Reply:
Yeah a lot of strings were copied wholesale from [edk2-platforms](https://github.com/tianocore/edk2-platforms/tree/master/Features/Intel/UserInterface/UserAuthFeaturePkg) without a thorough review. At some point I wanted to go through all of EDK2 and rewrite user facing strings, but that would break automated tests :cry:

Reply:
For completeness, fixed  in https://github.com/Dasharo/edk2/pull/152/commits/5218815922893eb9d914177e18df0d56f10f090d

Reply:
> At some point I wanted to go through all of EDK2 and rewrite user facing strings, but that would break automated tests

Is there an issue on this?

Do not worry that much about tests, if it is just change of string, this can be simply adjusted in tests.